packages/mapviewer/src/router/storeSync/storeSync.config.js

```diff
@@ -35,7 +35,11 @@ const storeSyncConfig = [
         keepInUrlWhenDefault: true,
         valueType: String,
         validateUrlInput: (store, query) =>
-            getStandardValidationResponse(query, SUPPORTED_LANG.includes(query), 'lang'),
+            getStandardValidationResponse(
+                query,
```

Comment:
you could directly `.toLowerCase()` the query here, no need to change the store then

Reply:
this is the validation, not the dispatcher.

If we wanted the dispatch to handle the case, we could either: 
 - create a new "complex" param for lang just to override the dispatch function
 - add a "isCaseSensitive" param to `AbstractParamConfig` which would then be used to determine if the value given is case sensitive, should be lowercased or should be uppercase
 
 If that's okay with you, I would go with the first solution.

Reply:
nah, nevermind, let's keep it the way you did it then.